spring-xd-dirt/src/main/java/org/springframework/xd/dirt/server/StreamRuntimePropertiesProvider.java

```diff
@@ -81,6 +81,10 @@ public RuntimeModuleDeploymentProperties propertiesForDescriptor(ModuleDescripto
 			if (hasPartitionKeyProperty(previousProperties)) {
 				properties.put("consumer." + BusProperties.PARTITION_INDEX, String.valueOf(moduleSequence - 1));
 			}
+			String minPartitionCount = previousProperties.get("producer." + BusProperties.MIN_PARTITION_COUNT);
```

Comment:
I realized we don't have test coverage for all these properties calculations. Created https://jira.spring.io/browse/XD-3143 for this.